docs/particles.md

```diff
@@ -134,6 +134,7 @@ The lifespan is passed down to all the descendants of a given `Particle`, if it
 Flame ships with a few built-in `Particle` behaviors:
 * The `TranslatedParticle`, translates its `child` by given `Offset`
 * The `MovingParticle`, moves its `child` between two predefined `Offset`, supports `Curve`
+* The `ConstantVelocityMovingParticle`, moves its `child` between two predefined `Offset` at a constant `velocity`, supports `Curve`
```

Comment:
how is this different than using MovingParticle with a linear curve?

Reply:
This is a good point, maybe this addition will just confuse the user?